src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,65 @@
+package shop;
+
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CartTest {
```

Comment:
I do not like your before and after hooks, but I think you added them for debugging